runtime/doc/eval.txt

```diff
@@ -1718,6 +1718,23 @@ v:swapcommand	Normal mode command to be executed after a file has been
 		example, when jumping to a tag the value is ":tag tagname\r".
 		For ":edit +cmd file" the value is ":cmd\r".
 
+				*v:t_TYPE* *v:t_bool* *t_bool-varialble*
```

Comment:
It's corrected by a later runtime patch, so ok (and probably preferred) to leave it. Thanks for pointing it out though @k-takata .